System/Posix/Env.hsc

```diff
@@ -34,9 +34,12 @@ import Foreign.C.String
 import Foreign.Marshal.Array
 import Foreign.Ptr
 import Foreign.Storable
-import Control.Monad
+import Control.Monad (forM_, unless)
 import Data.Maybe (fromMaybe)
 import System.Posix.Internals
+#if HAVE_CLEARENV
+import Control.Monad (void)
```

Comment:
Explicit import lists may be helpful, but IMHO do not justify introduction of CPP.

Reply:
This particular CPP is already in use in below, and its use here IMHO documents the reason why (void) is separately included. Otherwise we get warnings about redundant imports...  Are you strongly against this?

Reply:
Done in upcoming push